app/templates/scenario-builder.html

```diff
@@ -49,8 +49,9 @@ <h3 class="panel-title">{{section.name}}</h3>
                           <div class="form-group">
                             <label class="control-label" for="run-env">
                               Run environment
-                              <span class="glyphicon glyphicon-info-sign" data-toggle="tooltip" data-placement="right" title="Specifies where the simulations are hosted."></span>
```

Comment:
A data-toggle is often put on a `<button>` or `<a>`. Would that be a better solution?